Makefile

```diff
@@ -65,7 +80,7 @@ test-integration:
 # Build manager binary
 .PHONY: manager
 manager: generate vet
-	CGO_ENABLED=0 GO111MODULE=on GOOS=linux GOARCH=amd64 go build -a -o manager main.go
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
```

Comment:
Maybe we should detect os/arch of the system here? [Ready to use clause](https://github.com/ory/k8s/blob/master/Makefile#L1)